pkg/services/accesscontrol/evaluator/evaluator_test.go

```diff
@@ -0,0 +1,65 @@
+package evaluator
+
+import (
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtractPermission(t *testing.T) {
+	targetPermission := "permissions:create"
+	userPermissions := []*accesscontrol.Permission{
+		{
+			Action: "permissions:create",
+			Scope:  "teams:*/permissions:*",
+		},
+		{
+			Action: "permissions:remove",
+			Scope:  "permissions:*",
+		},
+	}
+	expectedScopes := map[string]struct{}{
+		"teams:*/permissions:*": {},
+	}
+	ok, scopes := extractPermission(userPermissions, targetPermission)
+	assert.True(t, ok)
+	assert.Equal(t, scopes, expectedScopes)
```

Comment:
The expectation comes first with `assert.Equal`:
```suggestion
	assert.Equal(t, expectedScopes, scopes)
```